styles.css

```diff
@@ -0,0 +1,1006 @@
+* {
+  box-sizing: border-box;
+  font-family: 'Lato', sans-serif;
+}
+
+html {
+  font-size: 16px;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.overFlow {
+  overflow: hidden;
+}
+
+.navbar-menu,
+.navbar-social {
+  display: none !important;
+}
+
+.bar {
+  display: block;
+  width: 2rem;
+  height: 0.3rem;
+  margin: 0.3rem;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease;
+  background-color: #424141;
+}
+
+.hamburger {
+  margin: 2rem;
+}
+
+.intro-main {
+  background-image: url(./images/BackgroundHomePageMobile.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position-y: -5rem;
+  z-index: 1;
+}
+
+.intro-about {
+  background-image: url(./images/BackgroundHomePageMobile.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.intro {
+  width: 90%;
+  margin: 0 5% 0 5%;
+  padding: 4rem 0 7rem;
+}
+
+/* home page */
+
+.greet {
+  color: #ec5242;
+  font-size: 1.5rem;
+  font-weight: 400;
+}
+
+.title {
+  background-image: url(./images/text-image.jpeg);
+  background-size: contain;
+  color: transparent;
+  -webkit-text-fill-color: transparent;
+  background-clip: border-box;
+  -webkit-background-clip: text;
+  font-size: 2rem;
+  font-weight: 900;
+  line-height: 2.5rem;
+  letter-spacing: 0.1rem;
+}
+
+.description {
+  color: #272a31;
+  font-size: 0.9rem;
+  background-color: #f0f0f0;
+  padding: 1rem;
+  border: 0.125rem solid #fff;
+}
+
+.date {
+  font-size: 1.8rem;
+  font-weight: 800;
+  color: #df7b2e;
+}
+
+.place {
+  font-size: 1.9rem;
+  color: #df7b2e;
+}
```

Comment:
- [ ] I really appreciate your efforts and patience ❤️. You did things according to your previous code reviewer's request but still, you have a small issue. As a code reviewer, our mission is to make a fully-fledged full-stack developer from all aspects. Your design should look like Behance because it will make your detail-oriented developer who can do complete work without any problem and according to user requirements. So please look at the below comparison table and try to understand what you need to change.

### Design Comparison

<table>
<tr>
<th>Your Design</th>
<th>Behance Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-05-25 09-54-25](https://github.com/gdvl777/CapstoneSmash/assets/66476329/6fbef2b3-3764-4a4b-bb58-b445c940d247)


</td>

<td>

![image](https://github.com/gdvl777/CapstoneSmash/assets/66476329/aa309ff3-7b30-465e-9d55-637747425850)

</td>
</tr>
</table>